include/glow/Partitioner/Partitioner.h

```diff
@@ -143,6 +143,10 @@ class Partitioner {
   /// The backend pointers.
   std::vector<Backend *> backends_;
 
+  /// The map between BackendKind and its name. It is used for generating debug
+  /// info.
+  std::map<BackendKind, std::string> backendName_;
```

Comment:
Does it need to be an ordered map?

Reply:
No. The order doesn't matter. We just want to get the name from the kind.

Reply:
Does it save much to have this map? Don't you have a Backend ptr you can just use getName on?

Reply:
@nickgg No. The backend ptr is local and only used in backendBasedPartition now. (Btw, the "backends_" ptrs are only used for users passing the "Mock" backends which are not registered in Glow. )